test/functional/ui/syntax_conceal_spec.lua

```diff
@@ -0,0 +1,401 @@
+            {1:∧}                                                    |
+                                                                 |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            :syn match dAmpersand '[&][&]' conceal cchar=∧       |
+          ]])
+        end)
+
+        it('&& Move Cursor Down', function() 
+          feed("ggj")
+          screen:expect([[
+            {1:∧}                                                    |
+            ^&&                                                   |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+                                                                 |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            :syn match dAmpersand '[&][&]' conceal cchar=∧       |
+          ]])
+        end)
+
+        it('&& Move Cursor to Bottom', function() 
+          feed("ggG")
+          screen:expect([[
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            {1:∧}                                                    |
+            ^                                                     |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            ~                                                    |
+            :syn match dAmpersand '[&][&]' conceal cchar=∧       |
+          ]])
+        end)
+      end) -- Multiple
+    end) -- Match 
+      
+    describe("Keyword", function()
```

Comment:
Perhaps you intend to add more tests here but otherwise this could be simplified to a single `it` block (`describe` is for grouping multiple related tests).